lib/generators/components/USAGE

```diff
@@ -0,0 +1,5 @@
+Description:
+    Generates a new component
+
+Example:
+    rails generate components:component [component_name]
```

Comment:
I think the namespace should be `govuk_publishing_components` like the rest of the gem.